addon.gypi

```diff
@@ -109,7 +109,7 @@
           '-luuid.lib',
           '-lodbc32.lib',
           '-lDelayImp.lib',
-          '-l"<(node_root_dir)/$(ConfigurationName)/<(node_lib_file)"'
+          '-l<(node_root_dir)/<(CONFIGURATION_NAME)/<(node_lib_file)'
```

Comment:
Sorry, may be i was not clear. What i mean is for `MsBuild`, this variable is declared [here](https://github.com/nodejs/node-gyp/blob/master/gyp/pylib/gyp/MSVSSettings.py#L426) which makes it available here as well as at other places. I am wondering, with this change if you build using `MsBuild`, what value will this variable resolve to?